src/apps/AppCenter/AppCenter.js

```diff
@@ -134,83 +122,55 @@ class AppCenter extends React.Component {
+      compactMode,
+      reposLoading,
+      onUpgradeAll,
+      canUpgradeOrg,
+    } = this.props
     const { upgradePanelOpened } = this.state
     const { activeTab, openedRepoName } = this.getLocation()
     const repos = this.getRepos()
     const currentRepo = openedRepoName && this.getRepoFromName(openedRepoName)
 
-    const navigationItems = [
-      'App Center',
-      ...(currentRepo ? [currentRepo.name] : []),
-    ]
-
     return (
       <React.Fragment>
-        <AppView
-          style={{ height: '100%' }}
-          appBar={
-            <Viewport>
-              {({ below, above }) => (
-                <AppBar
-                  tabs={
-                    openedRepoName ? null : (
-                      <div
-                        css={`
-                          margin-left: ${below('medium') ? '-14px' : '0'};
-                        `}
-                      >
-                        <TabBar
-                          items={SCREENS.map(screen => screen.label)}
-                          selected={activeTab}
-                          onChange={this.handleScreenChange}
-                        />
-                      </div>
-                    )
-                  }
-                >
-                  {below('medium') && navigationItems.length < 2 && (
-                    <MenuButton
-                      onClick={this.handleMenuPanelOpen}
-                      css={`
-                        position: relative;
-                        z-index: 2;
-                        margin-left: 0;
-                        margin-right: -24px;
-                      `}
-                    />
-                  )}
-                  <NavigationBar
-                    items={navigationItems}
-                    onBack={this.handleCloseRepo}
-                  />
-                  {canUpgradeOrg && !openedRepoName && (
-                    <UpgradeButton
-                      mode={below('medium') ? 'text' : 'strong'}
-                      onClick={onUpgradeAll}
-                      title="Upgrade all"
-                    >
-                      {below('medium') ? <IconUpgrade /> : 'Upgrade all'}
-                    </UpgradeButton>
-                  )}
-                </AppBar>
-              )}
-            </Viewport>
-          }
-        >
-          {activeTab === 0 &&
-            (reposLoading ? (
-              <EmptyBlock>Loading apps…</EmptyBlock>
-            ) : (
-              <InstalledApps
-                onOpenApp={this.handleOpenRepo}
-                onRequestUpgrade={this.handleOpenUpgradePanel}
-                openedRepoId={currentRepo && currentRepo.appId}
-                repos={repos}
+        <Header
+          primary="App center"
+          secondary={
+            canUpgradeOrg &&
+            activeTab !== 1 &&
+            !openedRepoName && (
+              <Button
+                mode="strong"
+                onClick={onUpgradeAll}
+                label="Upgrade all"
```

Comment:
cc @dizzypaty Perhaps we can change this to "Upgrade to 0.8" since it's not really "upgrade all" (it won't upgrade apps like TPS).

Reply:
@sohkai The 'Upgrade all' button should be displayed when there's more than 1 app contract to be upgraded. For this release, as the front-end will be automatically updated and users will be presented with the Camino modal showcase, I don't think we need to have an 'Upgrade to 0.8' button on the app center as it won't have any action attached (as far as I understood?)

Reply:
This button is displayed in the App Center if the organization is still a 0.6 organization (similar to the showcase banner).

In this case, the apps that are getting upgraded are only Finance, Tokens, Vault, and Voting. If you have other apps installed, from other developers (e.g. Autark), you won't be upgrading those apps, and hence why "upgrade all" is a bit confusing.

Reply:
Right! yep, the change you suggested ("Upgrade to 0.8") makes sense.

Reply:
PR [here](https://github.com/aragon/aragon/pull/940)